docs/userGuide/cliCommands.md

```diff
@@ -167,6 +167,48 @@ The caveat is that not building all pages during the initial process, or not reb
 
 <hr><!-- ========================================================================== -->
 
+### `archive` Command
+<br>
+
+**Format:** `markbind archive [options] [versionName] [archivePath]`
+
+**Alias:** `markbind ar`
+
+**Description:** Archives a version of the current site, which will not be affected by future changes to the site
+
```

Comment:
Perhaps can mention that a `version.json` will be created as well. Or, given the number of steps involved, perhaps can follow how the `serve` command is [described](https://markbind.org/userGuide/cliCommands.html#serve-command)